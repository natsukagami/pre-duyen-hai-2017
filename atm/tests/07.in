84 9
042680187040301847478187694278449409769918317190152567948986805840042652367969795217
