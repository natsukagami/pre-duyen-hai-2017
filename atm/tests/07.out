887431847478187694278449409769918317190152567948986805840042652367969795217
